src/components/TradeAssetSearch/TradeAssetSearch.tsx

```diff
@@ -104,7 +106,9 @@ export const TradeAssetSearch: FC<TradeAssetSearchProps> = ({
   const handleSubmit = useCallback((e: FormEvent<unknown>) => e.preventDefault(), [])
 
   const popularAssets = useMemo(() => {
-    const unfilteredPopularAssets = popularAssetsByChainId?.[activeChainId] ?? []
+    const unfilteredPopularAssets = (popularAssetsByChainId?.[activeChainId] ?? []).filter(asset =>
+      isSwapper ? asset.chainId !== KnownChainIds.SolanaMainnet : true,
```

Comment:
q: why do we only want this condition for `isSwapper` ? Gut feel is other domains would benefit from the same filtering

Reply:
from dm: That component is for both regular send and receive asset search as well as for swapper. We support Solana for everything except swapper so just added the condition to filter accordingly. Just temp logic realistically